setup.py

```diff
@@ -60,6 +60,7 @@
         "urllib3~=1.26.17",
         "certifi>=2024.7.4,<2024.9.0",
         "packaging>=23.2,<25.0",
+        "markupsafe~=2.1.3"
```

Comment:
Why is this being added?

Reply:
this one I can remove...it was added to try an support python 3.8...but could not